src/components/Main.js

```diff
@@ -1,19 +1,55 @@
 import React, { Component } from 'react';
-import RegistrationForm from '../containers/RegistrationForm';
+import { Switch, Route, Redirect, withRouter } from 'react-router-dom'
+
+import { connect } from 'react-redux';
+
+import RegistrationForm from '../containers/form/RegistrationForm';
+import LogInForm from '../containers/form/LogInForm';
+import HomePage from '../containers/HomePage';
+
 
 import './Main.sass'
 
-export default class Main extends Component {
+class Main extends Component {
 
     render() {
         return (
             <main className='main'>
-                <div className="container">
-                    <div className="row">
-                        <RegistrationForm />
-                    </div>
-                </div>
+                <Switch>
+                    <Route exact path='/' render={ () => (
```

Comment:
Do we have any another way how to render, except writing render method exactly here?
Also, I would recommend you check differently if user authorized.